src/types/simulation.ts

```diff
@@ -167,8 +167,11 @@ export type Span = {
   attributes: SpanAttributes;
   dataset_id: number;
   duration: string;
+  durationMs: number;
```

Comment:
These are also computed on the client and not stored in the db similar to directive start_time_ms, need to figure out best practice for computed values